spec/features/visitor_views_profile_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "A Visitor views a profile" do
+  scenario "and sees recommendations" do
+    someone_elses_show = create(:television_show)
+    recommendation = create(:recommendation)
+    user = recommendation.user
```

Comment:
What do you think about `recommending_user` and being a bit more specific?

Really, so many 'm's in this app.